package.json

```diff
@@ -14,9 +14,17 @@
   "builders": "builders.json",
   "scripts": {
     "build": "tsc",
-    "build:schematics": "tsc -p tsconfig.schematics.json"
+    "build:schematics": "tsc -p tsconfig.schematics.json",
+    "pretty": "prettier --write .",
+    "precise-commits": "precise-commits"
```

Comment:
This dependency, although I did not know its existence, seems a bit overkill for this project. AFAICU it is a good fit for large scale enterprise projects. Do you think it should worth adding it?